doc/visualizations.rst

```diff
@@ -87,3 +87,4 @@ Display Objects
    metrics.DetCurveDisplay
    metrics.PrecisionRecallDisplay
    metrics.RocCurveDisplay
+   model_selection.LearningCurveDisplay
```

Comment:
For a reason that I don't understand, the [rendering of this section](https://output.circle-artifacts.com/output/job/3ae38aaa-40fd-453d-a0c9-0ac4c1374852/artifacts/0/doc/visualizations.html) shows a `LearningCurveDisplay` that was never computed!

Reply:
For info, the problem can be fixed by changing
```diff
- :context:
+ :context: close-figs
```
on line 24 of the `doc/visualizations.rst`